# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/text/reftest.list
# Commit: 2b42abbdb0df
# Full Hash: 2b42abbdb0df38f31dfa1178fe3b5f773f8e4812
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-20 21:58:21
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 2 - Performance and quality fixes for part 1. r=gfx-reviewers,nical
#   
#   * Add support for local scale factors to a surface, allowing it to
#     be rasterized in root coordinate space. This allows snapping to
#     work across surfaces where the surface transform is a fractional
# ==============================================================================

diff -r 9fa6f71111d0 -r 2b42abbdb0df gfx/wr/wrench/reftests/text/reftest.list
--- a/gfx/wr/wrench/reftests/text/reftest.list	Sun Feb 20 18:56:54 2022 +0000
+++ b/gfx/wr/wrench/reftests/text/reftest.list	Sun Feb 20 18:56:55 2022 +0000
@@ -81,5 +81,5 @@
 # most pixels are off by a small amount, but a few pixels on the edge vary by a lot, pushing up the fuzzy max-diff;
 # the main goal of the test is that everything is in the same place, at the same scale, clipped the same way,
 # despite 4x on-the-fly scale change.
-skip_on(android) fuzzy(205,26560) == raster_root_C_8192.yaml raster_root_C_ref.yaml
+skip_on(android) fuzzy(215,7600) == raster_root_C_8192.yaml raster_root_C_ref.yaml
 == subpx-bg-mask.yaml subpx-bg-mask-ref.yaml